### ISSUE TITLE
Debugging bar not showing in PHP >= 5.4.4 (error in info.panel.phtml)

### ISSUE THREAD

**Issue by Sharasuke** [nette/tracy#317]
Version: 2.5.2

### Bug Description
While getting it to work/debugging via xdebug in PHP5.4.45 I came across an error message:
message:"syntax error, unexpected '&'"
file:"/var/www/vhosts/.../httpdocs/vendor/tracy/tracy/src/Tracy/assets/Bar/info.panel.phtml"
line:53

This is in the require statement of the panel in DefaultBarPanel.php on line: 50
While setting a breakpoint in the anonymous function in ob_start you can see the error if you run error_get_last() in the debug console.

More info: https://secure.php.net/manual/en/control-structures.foreach.php 
"Before PHP 5.5.0, referencing $value is only possible if the iterated array can be referenced (i.e. if it is a variable). The following code works only as of PHP 5.5.0: "
But the required php version is still  >=5.4.4 as seen in packagist.

### Steps To Reproduce
Running: composer require tracy/tracy in a PHP 5.4.45 environment

### Expected Behavior
Debug bar should show up

### Possible Solution
Transforming
`foreach ([&$packages, &$devPackages] as &$items) {`
into
`foreach ([&$packages, &$devPackages] as $items) {`

Or making it work in 5.4 and 5.5
Or changing required PHP version

**Comment by milo** [comment]
Which Tracy version? Master branch target PHP 7.1, the 2.5.2 is PHP 5.4 compatible.

**Comment by Sharasuke** [comment]
Tracy version 2.5.2.
According to the php manual page I referenced:
src/Tracy/assets/Bar/info.panel.phtml:53 is not compatible with PHP version < 5.5.0

**Comment by milo** [comment]
I see. Caused by d1ec8a24e035d507c767ac450425b80fc38107d9. Probably fixable by:
```php
$tmp = [&$packages, &$devPackages];
foreach ($tmp as &$items) {...}

**Comment by dg** [comment]
Fixed, thanks

**Comment by MT-QD** [comment]
The same fix should be applied to the v2.4 branch because 7a0390c broke the 5.4.4 support there too
